src/triage/component/postmodeling/contrast/postmodeling_config_example.yaml

```diff
@@ -0,0 +1,34 @@
+# Postmodeling Configuration File
```

Comment:
Now that there is an example config directory, this should probably go in it:

project root/example/config